### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by Derimarg** [Derimarg/code-command#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Derimarg** [comment]
**Hi there! <img src="https://user-images.githubusercontent.com/1303154/88677602-1635ba80-d120-11ea-84d8-d263ba5fc3c0.gif" width="28px" alt="hi">**

This is gonna be our discussion’s place. Conversation with respect to ideas for features.
@ratalla816 @audryf @georgedmendoza  

- [ ] Keep track of ideas.
- [ ] Ask for help
- [ ] Ask for a review
- [ ] Ask for ui-styles
- [ ] Ask for new features
- [ ] Ask for images to use
- [ ] Ask for documentation description
- [ ] ASk for social media to use
- [ ] and more


Thanks for being part of this project, HAPPY CODING!

**Comment by audryf** [comment]
For the search/filter feature, instead of using an input search bar could I use 2 dropdown menus? One for category and one for difficulty? I can add those to the product data set and everything I believe this would make the backend part a little easier due to the vast differences in possible user inputs.

**Reply by Derimarg** [reply]
Yeah, we can try that!

**Reply by audryf** [reply]
Do we need to change the styled component to the drop down menus first, to do this?

**Reply by Derimarg** [reply]
You can do just a simple drop down component without style, and just make sure that is working properly, and then we can work with the style.
